testapi.pm

```diff
@@ -477,11 +462,8 @@ been matched at the time of the call.
 
 =cut
 
-sub match_has_tag {
-    my ($tag) = @_;
-    if ($last_matched_needle) {
-        return $last_matched_needle->{needle}->has_tag($tag);
-    }
+sub match_has_tag ($tag) {
```

Comment:
https://openqa.opensuse.org/tests/2359829#step/start_install/2 fails with

```
# Test died: Too many arguments for subroutine 'testapi::match_has_tag' at opensuse/tests/installation/start_install.pm line 89.
	testapi::match_has_tag("inst-overview-error-found", 0) called at opensuse/tests/installation/start_install.pm line 89
```

introduced in

```
commit cf919f2f6
Author: Dumitru Gutu <dgutu@suse.de>
Date:   Wed Feb 8 18:49:43 2017 +0100

    UEFI boot partition on raid1

diff --git a/tests/installation/start_install.pm b/tests/installation/start_install.pm
index 6ac47ff09..830bbb0e8 100644
--- a/tests/installation/start_install.pm
+++ b/tests/installation/start_install.pm
@@ -81,6 +81,7 @@ sub run() {
     else {
         sleep 2;    # textmode is sometimes pressing alt-i too early
         send_key $cmd{install};
+        wait_screen_change { send_key 'alt-o' } if match_has_tag('inst-overview-error-found', 0);
         while (check_screen([qw(confirmlicense startinstall)], 5)) {
             last if match_has_tag("startinstall");
             send_key $cmd{acceptlicense}, 1;
```

`git grep 'match_has_tag.*, [0-9]'` in os-autoinst-distri-opensuse reveals multiple other occurences *sigh*

-> https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14944

Reply:
Merged, fixed